README.md

```diff
@@ -1,37 +1,21 @@
-<p align="center">
-      <a href="https://github.com/SAP/fundamental-ngx" target="_blank" rel="noopener noreferrer">
-            <img src="https://raw.githubusercontent.com/SAP/fundamental-ngx/develop/assets/images/readme/temp_logo_sap_2.png" alt="Fundamental NGX logo">
-      </a>
-</p>
-
-<p align="center">
-      Fundamental NGX is a lightweight Angular component library for <a href="https://github.com/SAP/fundamental">SAP Fiori Fundamentals</a>.
-</p>
-
-<p align="center">
-      <a href="https://travis-ci.org/SAP/fundamental-ngx">
-            <img src="https://travis-ci.org/SAP/fundamental-ngx.svg?branch=develop" alt="Build Status">
-      </a>
-      <a href="https://www.npmjs.com/package/fundamental-ngx">
-            <img src="https://badge.fury.io/js/fundamental-ngx.svg" alt="npm version">
-      </a>
-      <a href="https://ui-fundamentals.slack.com">
-            <img src="https://img.shields.io/badge/slack-ui--fundamentals-blue.svg?logo=slack" alt="Slack Community">
-      </a>
-</p>
-
-
-<p align="center">
-:open_book:
-<strong><a href="https://sap.github.io/fundamental-ngx/docs/home">Component Documentation</a></strong>
-:open_book:
-</p>
+# Fundamental NGX
+
+[![npm version](https://badge.fury.io/js/fundamental-ngx.svg)](//www.npmjs.com/package/fundamental-ngx)
+[![Minified Size](https://badgen.net/bundlephobia/min/fundamental-ngx)](https://bundlephobia.com/result?p=fundamental-ngx)
+[![Minzipped Size](https://badgen.net/bundlephobia/minzip/fundamental-ngx)](https://bundlephobia.com/result?p=fundamental-ngx)
+[![Build Status](https://travis-ci.org/SAP/fundamental-ngx.svg?branch=master)](https://travis-ci.org/SAP/fundamental-ngx)
+[![Coverage Status](https://coveralls.io/repos/github/SAP/fundamental-ngx/badge.svg?branch=develop)](https://coveralls.io/github/SAP/fundamental-ngx?branch=develop)
+[![Slack](https://img.shields.io/badge/slack-ui--fundamentals-blue.svg?logo=slack)](https://ui-fundamentals.slack.com)
```

Comment:
your badges should probably refer to the same branch, be it master, or develop.

Reply:
done